docs/reference/terminal49/terminal49.v1.json

```diff
@@ -5359,6 +5391,13 @@
               "number": {
                 "type": "string"
               },
+              "ref_numbers": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "nullable": true
```

Comment:
Does the serializer return null or an empty array if it's not there?

Reply:
fixed ace39829c980c4a2cbbe3110c49998a75ed77156

Reply:
fixed b6a23815cfb07525018946306c58ed7435242269